packages/discord.js/typings/index.d.ts

```diff
@@ -2836,16 +2836,14 @@ export class GuildChannelManager extends CachedManager<Snowflake, GuildBasedChan
   ): Promise<MappedGuildChannelTypes[T]>;
   /** @deprecated See [Self-serve Game Selling Deprecation](https://support-dev.discord.com/hc/en-us/articles/4414590563479) for more information */
   public create(
```

Comment:
You need update `GuildChannelCreateOptions` too (add name property).
Same for `ChannelData` and `RoleData` but you need add reason